### PR TITLE
Adding "Our World"

### PR DESCRIPTION
**Pull request by agude**
Starting to get the text for our world written.

### CODE REVIEW

File: chapters/02-world-building.md [hunk 1]
```diff
@@ -55,13 +47,33 @@ change them during character creation, or even after the story is underway!
...
+<!--Do we need more of an intro? Like "We'll build an example world, example
+characters, and use them to show you how to play..." ?-->
+
+Our players decide to be air pirates in a steampunk fantasy world. They want
+magic, sword fights, morally gray adventurers in a world of intrigue where
+they can never be sure if they're the conman or the mark. They write down the
+following two Aspects:
+
+- Genre: \aspect{Steam-Powered Swashbuckling on the Seven Seas}
+- Trouble: \aspect{A Puppet-master is Pulling All the Strings}
```
Comment (charles-uno) [inline chapters/02-world-building.md:74]: Gonna add a build check for capitalization after dashes
```suggestion
- Trouble: \aspect{A Puppet-Master is Pulling All the Strings}
```

File: chapters/02-world-building.md [hunk 2]
```diff
@@ -55,13 +47,33 @@ change them during character creation, or even after the story is underway!
...
 
-The GM will often create these Aspects as they describe the scene to the
-players. Some examples: \aspect{Alien Bar Where The Authorities Dare Not
-Tread}, \aspect{Mob's Docks At Midnight}, or \aspect{Ancient Fortification
-Surrounded by Orcs}.
+The GM will often create these aspects as they describe the scene to the
+players. Some examples:
 
-Just as with setting Aspects, don't be shy about updating Scene Aspects if the
+- \aspect{Ancient Fortification Surrounded by Orcs}
```
Comment (charles-uno) [inline chapters/02-world-building.md:55]: Don't love that these are all places. Can we get at least one that's stuff happening? Maybe 
```suggestion
- \aspect{An Army of Orcs Arrives at the Wall}
```

File: chapters/02-world-building.md [hunk 3]
```diff
@@ -55,13 +47,33 @@ change them during character creation, or even after the story is underway!
...
+- \aspect{Rust-bucket Hovercraft in the Real World}
```
Comment (charles-uno) [inline chapters/02-world-building.md:56]:
```suggestion
- \aspect{Rust-Bucket Hovercraft in the Real World}
```

File: chapters/02-world-building.md [hunk 4]
```diff
@@ -55,13 +47,33 @@ change them during character creation, or even after the story is underway!
...
+- \aspect{Alien Bar Where The Authorities Dare Not Tread}
+
+Just as with setting aspects, don't be shy about updating Scene aspects if the
 story calls for it. Remember: aspects are always true! <!-- TODO: I don't
 think we've ever said this before.-->
+
+## Our World
```
Comment (charles-uno) [inline chapters/02-world-building.md:63]: I was thinking of doing examples inline, using the block quote environment. Do you prefer sections instead?

File: chapters/02-world-building.md [hunk 5]
```diff
@@ -55,13 +47,33 @@ change them during character creation, or even after the story is underway!
...
-Surrounded by Orcs}.
+The GM will often create these aspects as they describe the scene to the
+players. Some examples:
 
-Just as with setting Aspects, don't be shy about updating Scene Aspects if the
+- \aspect{Ancient Fortification Surrounded by Orcs}
+- \aspect{Rust-bucket Hovercraft in the Real World}
+- \aspect{Alien Bar Where The Authorities Dare Not Tread}
+
+Just as with setting aspects, don't be shy about updating Scene aspects if the
```
Comment (charles-uno) [inline chapters/02-world-building.md:59]:
```suggestion
Just as with setting aspects, don't be shy about updating scene aspects if the
```
?

File: chapters/02-world-building.md [hunk 6]
```diff
@@ -55,13 +47,33 @@ change them during character creation, or even after the story is underway!
...
 story calls for it. Remember: aspects are always true! <!-- TODO: I don't
 think we've ever said this before.-->
+
+## Our World
+
+<!--Do we need more of an intro? Like "We'll build an example world, example
+characters, and use them to show you how to play..." ?-->
+
+Our players decide to be air pirates in a steampunk fantasy world. They want
```
Comment (charles-uno) [inline chapters/02-world-building.md:68]:
```suggestion
Our players decide to be sky pirates in a steampunk fantasy world. They want
```

File: chapters/02-world-building.md [hunk 7]
```diff
@@ -55,13 +47,33 @@ change them during character creation, or even after the story is underway!
...
+magic, sword fights, morally gray adventurers in a world of intrigue where
```
Comment (charles-uno) [inline chapters/02-world-building.md:69]:
```suggestion
magic, sword fights, morally gray adventures in a world of intrigue where
```
I think our genre is goofy enough that we might need to scale back how serious the setting claims to be

File: chapters/02-world-building.md [hunk 8]
```diff
@@ -55,13 +47,33 @@ change them during character creation, or even after the story is underway!
...
+they can never be sure if they're the conman or the mark. They write down the
+following two Aspects:
+
+- Genre: \aspect{Steam-Powered Swashbuckling on the Seven Seas}
+- Trouble: \aspect{A Puppet-master is Pulling All the Strings}
+<!-- Trying to have a "ambiguous" aspect which we'll let the GM fill in the
+details of. -->
+
+In the next chapter we'll build their characters; from then on we'll use that
```
Comment (charles-uno) [inline chapters/02-world-building.md:78]: "our world" vs "their characters"

**Review by charles-uno — APPROVED** [review]
Quibbles, but it mostly looks good. Big question is formatting for examples. I was thinking blockquotes inline. But that's easy enough to change later

File: chapters/02-world-building.md (hunk 4 above)
Comment (agude) [reply chapters/02-world-building.md:63]: Let's do quotes!

File: chapters/02-world-building.md (hunk 7 above)
Comment (agude) [reply chapters/02-world-building.md:69]: Sounds good!

**Pull request merged**